src/api/app/controllers/webui/users/tokens_controller.rb

```diff
@@ -57,6 +59,25 @@ def destroy
     redirect_to tokens_url
   end
 
+  # method for the view
```

Comment:
not possible to move it to a new controller, subcontroller from TokensController with just some CRUD methods, instead such custom method names?

Reply:
Already on it :)